Math/isPrime.js

```diff
@@ -0,0 +1,16 @@
+// Whether the given number is a prime nujmber or not
```

Comment:
Looping till input number to find out whether it is prime or not is not optimal, right now it takes O(n) time,
Could you add  O(sqrt(n)) solution under same file with different function name